pipelines/azure-pipelines.load-test.yml

```diff
@@ -13,13 +12,23 @@ variables:
 
 steps:
 
-- script: |
-    az login --service-principal --username $(ARM_CLIENT_ID) --password $(ARM_CLIENT_SECRET) --tenant $(ARM_TENANT_ID)
-    az account set --subscription $(ARM_SUBSCRIPTION_ID)
-  displayName: 'SETUP: Login on Azure CLI'
+- task: AzureKeyVault@1
+  inputs:
+    azureSubscription: $(TF_VAR_SERVICE_CONNECTION_NAME)
+    KeyVaultName: $(TF_VAR_KEY_VAULT_NAME)
+    SecretsFilter: 'acr-secret,arm-client-secret'
+  displayName: 'Get secrets from Azure Key Vault'
+
+- task: AzureCLI@2
+  displayName: 'Login to Azure Container Registry'
```

Comment:
So this can be renamed to `SETUP: Validate JMeter Docker Image`

Reply:
done